### DISCUSSION TITLE
Conflict after editing a page which can't be resolved

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#3685] [Error / Bug Report]
**Describe the bug**
Conflict after editing a page which can't be resolved 

**To Reproduce**
1. Go to a wiki page
2. Edit some text
3. Save it
4. Click conflict button
5. Select any version
6 Get error

**Expected behavior**
Page saves normally

**Screenshots**
![Screenshot_20200801_165159](https://user-images.githubusercontent.com/36996396/89103024-55f9ec00-d417-11ea-94ba-c499512f5e36.png)

**Host Info (please complete the following information):**
 - OS: Docker
 - Wiki.js version: 2.4.107
 - Database engine: postgresql docker image

**Additional context**
Docker logs - https://del.dog/inifyphynu.txt

**Comment by bxjw** [comment]
Also seeing this issue, with Wiki.js version 2.5.170, and I believe we started seeing the same issue over the summer with some version as well. The page does actually seem to save properly if one closes the edit after choosing a conflict version, but will not acknowledge it and will continue to flag a conflict.
